microsoft-azure-sb/3.0/modules/ROOT/pages/azure-service-bus-connector-studio.adoc

```diff
@@ -78,15 +78,15 @@ To add an operation for Azure Service Bus Messaging, follow these steps:
 
 When you configure a connector, it’s best to configure a global element that all instances of that connector in the app can use. Configuring a global element requires you to provide the authentication credentials the connector uses to access the target Azure Service Bus Messaging system. 
 
-Azure Service Bus Messaging supports SAS (Shared Access signature) authentication.
+Azure Service Bus Connector supports SAS (Shared Access signature) authentication.
 
 To configure the global element for Azure Service Bus Connector, follow these steps:
 
 . Select the name of the connector in the Studio canvas.
 . In the configuration screen for the operation, click the plus sign (+) next to the *Connector configuration* field to access the global element configuration fields.
 . On the *General* tab, configure the following required fields for SAS authentication:
 .. Enter the Azure *Service namespace*. +
-The service namespace is the unique name assigned to the namespace at the time of creation. It is the host-name portion of the URL, for example: 
+The service namespace is the unique name assigned to the namespace when it was created and is the host-name portion of the URL, for example: 
```

Comment:
"The ...namespace is the...name assigned to the namespace"? This does not make sense. Please clarify.

Reply:
It does make sense. Namespace is an object and you need to name it.